attributes/default.rb

```diff
@@ -1,10 +1,12 @@
+default['mailcatcher']['ruby_version'] = '1.9.3-p545'
+
 default['mailcatcher']['http_ip'] = '0.0.0.0'
 default['mailcatcher']['http_port'] = 1080
 
 default['mailcatcher']['smtp_ip'] = '127.0.0.1'
 default['mailcatcher']['smtp_port'] = 1025
 
-default['mailcatcher']['bin'] = 'mailcatcher'
+default['mailcatcher']['bin'] = '/opt/rbenv/shims/mailcatcher'
```

Comment:
Why do you hard-code the path to the binary? Shells can locate it based on the `PATH` environment variable and it is more flexible if someone has a different path to rbenv install.